### PR TITLE
Add connection status

### PR DESCRIPTION
**Pull request by Or-Geva**
- [x] All [tests](https://github.com/jfrog/jfrog-vscode-extension#building-and-testing-the-sources) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----
The connection status to Xray is added in this PR.

Screen Shot 2022-08-18 at 21 16 28

### CODE REVIEW

File: src/main/commands/commandManager.ts [hunk 1]
```diff
@@ -287,6 +288,22 @@ export class CommandManager implements ExtensionComponent {
         }
     }
 
+    private async showConnectionStatus() {
+        if (this._connectionManager.areXrayCredentialsSet()) {
```
Comment (yahavi) [inline src/main/commands/commandManager.ts:292]: Let's do the same for Artifactory connection

File: src/main/commands/commandManager.ts [hunk 2]
```diff
@@ -287,6 +288,22 @@ export class CommandManager implements ExtensionComponent {
...
+            vscode.window.showInformationMessage(this.connectedMessage());
+        } else {
+            vscode.window.showErrorMessage('No connection to JFrog Xray');
+        }
+    }
+
+    private connectedMessage(): string {
+        let message: string = 'Connected To Xray ' + this._connectionManager.xrayUrl;
```
Comment (yahavi) [inline src/main/commands/commandManager.ts:300]:
```suggestion
        let message: string = 'Connected To JFrog Xray ' + this._connectionManager.xrayUrl;
```

File: src/main/connect/connectionManager.ts [hunk 3]
```diff
@@ -47,25 +47,35 @@ export class ConnectionManager implements ExtensionComponent {
...
     private _password: string = '';
     private _xrayUrl: string = '';
     private _rtUrl: string = '';
     private _url: string = '';
+    private _xrayVersion: string = '';
 
-    constructor(private _logManager: LogManager) {}
+    constructor(private _logManager: LogManager) {
+        this._statusBar = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right);
+        this._statusBar.tooltip = 'JFrog Xray connection details status';
```
Comment (yahavi) [inline src/main/connect/connectionManager.ts:62]:
```suggestion
        this._statusBar.tooltip = 'JFrog connection details';
```

File: src/main/connect/connectionManager.ts [hunk 4]
```diff
@@ -47,25 +47,35 @@ export class ConnectionManager implements ExtensionComponent {
...
+        this._statusBar.text = '🐸 JFrog Status';
```
Comment (yahavi) [inline src/main/connect/connectionManager.ts:63]: Please notice the space between the emoji and the text
```suggestion
        this._statusBar.text = '🟢 JFrog';
```
```suggestion
        this._statusBar.text = '🔴 JFrog';
```

**Review by yahavi — CHANGES_REQUESTED** [review]
Looks good!

**Review by Or-Geva — COMMENTED** [review]
@yahavi Fixes

**Review by yahavi — APPROVED** [review]
LGTM

**Pull request merged**